source/_components/plugwise.markdown

```diff
@@ -0,0 +1,164 @@
+---
+title: "Plugwise"
+description: "Plugwise Climate integration."
+logo: plugwise.png
+ha_category: Climate
+ha_iot_class: Local Polling
+ha_release: 0.98
+---
+
+This enables [Plugwise](https://plugwise.com) [Anna](https://www.plugwise.com/en_US/products/anna) thermostats to be integrated. This integration talks locally to your **Smile** interface, you will need its password and IP address.
+
+The password can be found on the bottom of your Smile, it should consist of 6 characters. To find your IP address use the Plugwise App: 
+ - Open the Plugwise App and choose the 'Settings'-icon (&#9776;) and choose 'HTML-interface'. 
+ - Go to the (lower) 'Settings'-icon (&#9776;) and choose 'Preferences'. 
+ - Choose 'System' then 'Networking' and your IP address will be shown.
+
+## Configuration
+
+You have to add the following to your configuration.yaml file.
+
+```yaml
+# Minimal configuration.yaml entry
+climate:
+  - platform: plugwise
+    password: your_short_id # required, the ID on the smile (some string of 6 characters)
+    host: local_ip_address  # required, the IP-address of your smile
+```
+
+{% configuration %}
+password:
+  description: Your Smile ID (located on the bottom of the Smile, not the Anna)
+  required: true
+  type: string
+host:
+  description: The IP address of your Smile. 
+  required: true
+  type: string
+name:
+  description: The name of your thermostat, i.e. "Anna"
+  required: false
+  type: string
+  default: "Plugwise Thermostat"
+username:
+  description: Should you ever need to change this, you can
+  required: false
+  type: string
+  default: smile
+port:
+  description: When having a custom setup, you can change the port number
+  required: false
+  type: integer
+  default: 80
+min_temp:
+  description: If you want to adjust the lower boundary, the integration will not allow temperatures below the set value.
+  required: false
+  type: integer
+  default: 4
+max_temp:
+  description: If you want to adjust the upper boundary, the integration will not allow temperatures above the set value.
+  required: false
+  type: integer
+  default: 30
+{% endconfiguration %}
+
+### Full configuration example
+
+```yaml
+# Example configuration.yaml entry to show only devices at your vacation and primary homes
+climate:
```

Comment:
Remove all comments, they should be part of the config section.
Make sure place holders are in all caps.

Reply:
Is this in-line with what you wanted?

Reply:
Yep, that is how it should be done 👍